### PR TITLE
Remove unneeded transmute calls

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Part of https://github.com/gtk-rs/gir/pull/896.

cc @sdroege @EPashkin

### CODE REVIEW

**Comment by EPashkin** [comment]
👍

**Comment by EPashkin** [comment]
Again only apveyor on CI :(

**Comment by sdroege** [comment]
Oh no, travis is unhappy

```
git diff -R --exit-code
diff --git b/src/auto/versions.txt a/src/auto/versions.txt
index c999e37..ec2c1de 100644
--- b/src/auto/versions.txt
+++ a/src/auto/versions.txt
@@ -1,2 +1,2 @@
 Generated by gir (https://github.com/gtk-rs/gir @ 997cbcb)
-from gir-files (https://github.com/gtk-rs/gir-files @ 480c117)
+from gir-files (https://github.com/gtk-rs/gir-files @ 22748fa)
Makefile:16: recipe for target 'regen_check' failed
make: *** [regen_check] Error 1
```

**Comment by GuillaumeGomez** [comment]
Updated.

File: src/lib.rs [hunk 1]
```diff
@@ -81,7 +81,6 @@ pub use auto::*;
 
 pub mod prelude;
 
-#[allow(clippy::transmute_ptr_to_ref)]
```
Comment (sdroege) [inline src/lib.rs:84]: Probably needs the same change in glib then?

Comment (GuillaumeGomez) [reply src/lib.rs:84]: Indeed!

**Pull request merged**